api/MonitorizareVot.Ong/src/MonitorizareVot.Domain.Ong/Models/USROngContextExtensions.cs

```diff
@@ -0,0 +1,231 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using ExcelDataReader;
+using ExcelDataReader.Exceptions;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using MonitorizareVot.Domain.Ong.Models;
+using MonitorizareVot.Domain.Ong.ValueObjects;
+
+
+namespace MonitorizareVot.Domain.Ong
+{
+    public static class USROngContextExtensions
+    {
+
+        private static List<Sectiune> _sectiuni = new List<Sectiune>
+        {
+                new Sectiune { IdSectiune = 1, CodSectiune = "A", Descriere = "Deschidere" }
+        };
+
+       
+  
+        public static void EnsureUSRSeedData(this OngContext context)
+        {
+
+            var intrebariExcel = ExcelParser.GetIntrebariObservatorFromFile();
+            List<Optiune> optiuni = GetOptiuni(intrebariExcel);
+            List<Intrebare> intrebari = GetIntrebariObservator(intrebariExcel, optiuni);
+
+            if (!context.AllMigrationsApplied())
+                return;
+
+            using (var tran = context.Database.BeginTransaction())
+            {
+                context.DataCleanUp();
+                context.SeedVersions();
+                context.SeedJudete();
+                context.SeedSectiune();
+                context.SeedOptiuni(optiuni);
+                context.SeedIntrebariObservator(intrebari);
+
+
+                tran.Commit();
+            }
+        }
+
+
+        private static void SeedIntrebariObservator(this OngContext context, List<Intrebare> intrebari)
+        {
+
+            if (context.Intrebare.Any())
+                return;
+            
+            context.Intrebare.AddRange(intrebari);
+            context.SaveChanges();
+        }
+
+
+
+        private static void SeedJudete(this OngContext context)
+        {
+            if (context.Judet.Any())
+                return;
+
+            context.Judet.AddRange(
+                new Judet { IdJudet = 0, CodJudet = "AB", Nume = "ALBA" },
```

Comment:
⚠️ IdJudet trebuie sa inceapa de la 1. asta a ramas aici in cod intr-un mod bizar :)